dev-tools/validation/test_adr_review_status.py

```
@@ -0,0 +1,853 @@
+    path.write_text(
+        path.read_text(encoding="utf-8") + f"\n{row}\n",
+        encoding="utf-8",
+    )
+
+
+def replace_once(text: str, old: str, new: str) -> str:
+    occurrences = text.count(old)
+    if occurrences != 1:
+        raise AssertionError(
+            f"expected exactly one occurrence of {old!r}, found {occurrences}"
+        )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)